src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -519,14 +519,14 @@ void MapSelect_PrintMenu(MapSelectState* this, GfxPrint* printer) {
     for (i = 0; i < 20; i++) {
         GfxPrint_SetPos(printer, 9, i + 4);
 
-        scene = (this->topDisplayedScene + i + this->count) % this->count;
-        if (scene == this->currentScene) {
+        sceneId = (this->topDisplayedScene + i + this->count) % this->count;
+        if (sceneId == this->currentScene) {
```

Comment:
these are actually not scene ids, just indices into z_select's `sScenes`

(probably should be renamed to "map" or "entrances" as a whole eventually at a later time)

Reply:
Sounds good. Just reverted this file for the time being.